awx_collection/tests/integration/targets/tower_credential/tasks/main.yml

```diff
@@ -210,10 +353,14 @@
     description: An example Vault credential
     vault_password: secret-vault
   register: result
+  ignore_errors: True
```

Comment:
So here, it is creating an SSH credential that _also_ has a vault password, and that is no longer allowed by the API 👍 

you should still be able to create a vault credential, and the prior task does exactly this.